workers/tests/integrationtests/test_indicator_mapping_saturation.py

```diff
@@ -57,6 +58,26 @@ def test_cannot_convert_nan_error(self):
         indicator.calculate()
         indicator.create_figure()
 
+    @oqt_vcr.use_cassette()
+    def test_avoiding_nan_in_sigmoid_curve(self):
+        """In some areas which contain mapped features, but mapping has happened in very
+        few contributions, sometimes not all curves could be calculated correctly and
+        those were chosen as the best-fitting curve. As a result, the output contained
+        nan-values instead of data or None values. This test checks the fix which avoids
+        choosing a curve containing nan values"""
```

Comment:
I would try to write pep conform docstrings: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. 

```suggestion
        """Test for NaN values of the `saturation` attribute.
        
        In some areas which contain mapped features, but mapping has happened in very
        few contributions, sometimes not all curves could be calculated correctly and
        those were chosen as the best-fitting curve. As a result, the output contained
        nan-values instead of data or None values. This test checks the fix which avoids
        choosing a curve containing nan values.
        """
```